clients/shared/table_config.go

```diff
@@ -90,35 +80,41 @@ func (g GetTableCfgArgs) GetTableConfig() (*types.DwhTableConfig, error) {
 			return nil, err
 		}
 
-		row := make(map[string]string)
+		row := make(map[string]*string)
```

Comment:
Could we just store an empty string when we get a back a `nil` rather than use pointers?

Reply:
Sure